client/src/main/scala/grasshopper/client/parser/model/ParsedAddress.scala

```diff
@@ -1,14 +1,8 @@
 package grasshopper.client.parser.model
 
-case class AddressPart(
-  addressNumber: String,
-  city: String,
-  state: String,
-  streetName: String,
-  zip: String
-)
-case class ParsedAddress(input: String, parts: AddressPart)
-
 object ParsedAddress {
-  def empty: ParsedAddress = ParsedAddress("", AddressPart("", "", "", "", ""))
+  def empty: ParsedAddress = ParsedAddress("", List.empty)
 }
+
+case class AddressPart(`type`: String, value: String)
```

Comment:
@hkeeler I usually avoid `type` since it's a reserved word and it causes this weird notation in scala. I think this is mostly fine, but at some point it might be worth discussing style for this sort of thing (many of these issues boil down to personal preference). Something like `partType` avoids this issue and reads almost the same